Source/ARTRealtime+Private.h

```diff
@@ -113,6 +116,10 @@ NS_ASSUME_NONNULL_BEGIN
 // Message sending
 - (void)send:(ARTProtocolMessage *)msg sentCallback:(nullable ARTCallback)sentCallback ackCallback:(nullable ARTStatusCallback)ackCallback;
 
++ (double)backoffCoefficientWithNumber:(NSInteger)number;
++ (double)jitterDelta;
++ (double)jitterCoefficient;
```

Comment:
Perhaps the naming could make it clear that this is doing something and isn't just returning a static value – maybe `generateRandomJitterCoefficient` or at least `generateJitterCoefficient`?

Reply:
https://github.com/ably/ably-cocoa/pull/1520/commits/171efb801fb805bb3d47064c713221affaf6405a